upload/install/cli_install.php

```diff
@@ -123,6 +123,12 @@ public function index(): void {
 		$this->response->setOutput($output);
 	}
 
+	/**
+	 * Install
+	 *
+	 * @param mixed   $argv
```

Comment:
Looks like this is actually supposed to be array<int, string>, I corrected it in a follow up